frontend/src/pages/auth/styles.module.scss

```
@@ -51,6 +48,23 @@
   font-size: 83px;
 }
 
+@media (width > 768px) and (width < 1023px) {
```

let's do it this way, check that nothing has changed
```suggestion
@media (width < 1023px) {
```